zutil_p.h

```diff
@@ -5,15 +5,23 @@
 #ifndef ZUTIL_P_H
 #define ZUTIL_P_H
 
-#ifdef __APPLE__
+#if defined(HAVE_POSIX_MEMALIGN) && !defined(_POSIX_C_SOURCE)
+#  define _POSIX_C_SOURCE 200112L
```

Comment:
I could have optionally set `_POSIX_C_SOURCE` in the build system, but I wanted to keep that localized to this file. I can always change it if needed.

Reply:
In some cases `_POSIX_C_SOURCE` is already set in build system, so this can fail if the previous value is too low.

Reply:
Fixed.